File: core/src/androidTest/mock/com/mobilesolutionworks/android/controller/test/basic/SchedulerTestActivity.java
//package com.mobilesolutionworks.android.controller.test.basic;
//
//import android.os.Bundle;
//import android.support.annotation.Nullable;
//import android.support.v4.app.Fragment;
//import android.support.v7.app.AppCompatActivity;
//import android.view.View;
//
//import com.mobilesolutionworks.android.controller.test.R;
//
///**
// * Created by yunarta on 9/3/17.
// */
//
//public class SchedulerTestActivity extends AppCompatActivity {
//
//    @Override
//    protected void onCreate(@Nullable Bundle savedInstanceState) {
//        super.onCreate(savedInstanceState);
//        setContentView(R.layout.fragment_host_activity);
//        findViewById(R.id.button).setOnClickListener(new View.OnClickListener() {
//            @Override
//            public void onClick(View v) {
//                getSupportFragmentManager().beginTransaction()
//                        .addToBackStack("back-stack")
//                        .replace(R.id.fragment_container, new Fragment(), "stack")
//                        .commit();
//            }
//        });
//
//        if (savedInstanceState == null) {
//            getSupportFragmentManager().beginTransaction()
//                    .replace(R.id.fragment_container, new SchedulerTestFragment(), "root")
//                    .commitNow();
//        }
//    }
//}

File: test-app/src/main/kotlin/com/mobilesolutionworks/android/controller/test/basic/SchedulerTestActivity.kt
package com.mobilesolutionworks.android.controller.test.basic

import android.os.Bundle
import android.support.v4.app.Fragment
import android.support.v7.app.AppCompatActivity
import android.widget.Button
import com.mobilesolutionworks.android.controller.test.R

/**
 * Created by yunarta on 9/3/17.
 */

open class SchedulerTestActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.fragment_host_activity)
        findViewById<Button>(R.id.button).setOnClickListener {
            supportFragmentManager.beginTransaction()
                  .addToBackStack("back-stack")
                  .replace(R.id.fragment_container, Fragment(), "stack")
                  .commit()
        }

        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                  .replace(R.id.fragment_container, SchedulerTestFragment(), "root")
                  .commitNow()
        }
    }
}
